### PR TITLE
Update committer page

### PR DESCRIPTION
**Pull request by BryanCutler**
This PR updates the list of committers to add myself.

### CODE REVIEW

**Comment by BryanCutler** [comment]
Wahoooo!

**Review by HyukjinKwon — APPROVED** [review]
Wow

File: committers.md [hunk 1]
```diff
@@ -15,6 +15,7 @@ navigation:
 |Joseph Bradley|Databricks|
 |Felix Cheung|Microsoft|
 |Mosharaf Chowdhury|University of Michigan, Ann Arbor|
+|Bryan Cutler|IBM|
```
Comment (gatorsmile) [inline committers.md:18]: Congratulations!

**Comment by BryanCutler** [comment]
Thanks @gatorsmile and @HyukjinKwon , merged!

**Pull request merged**